package.json

```diff
@@ -14,15 +14,13 @@
     "test": "jest",
     "prepublishOnly": "npm run build"
   },
-  "dependencies": {
-    "lodash": "^4.17.21"
-  },
   "peerDependencies": {
     "zod": "~3.14.4"
   },
   "devDependencies": {
-    "@types/jest": "^27.4.1",
+    "lodash": "^4.17.21",
```

Comment:
My idea was to leave it as a devDependency so that someone can use it for ease of  development. And then once that's the working version he can go and implement it :thinking:. I was thinking of adding an eslint rule for it once we add a linter to start with